src/components/NFTBook/ItemCard.vue

```diff
@@ -286,6 +285,9 @@ export default {
       this.updateNFTOwners();
       try {
         const blockingPromises = [this.fetchISCNMetadata()];
+        if ([PRESET_TYPE.CAMPAIGN, PRESET_TYPE.SHELF].includes(this.preset)) {
+          blockingPromises.push(this.fetchNFTBookPriceByClassId(this.classId));
```

Comment:
Should catch `CLASS_ID_NOT_FOUND` here since throwing might affect other `blockingPromises` and book here are not guaranteed to be registered on API